### DISCUSSION TITLE
Hey Jeremy, I had a question about the fast mass spring cloth system you have included in your software. (Description)

### DISCUSSION THREAD

**Discussion by aapatre** [huxingyi/dust3d#103] [Q&A]
CMake is unable to detect freeglut even though I had hardcoded/set the path in the cmakelist file. I'm using windows 10.
Can you please share how you built the code? :)

**Comment by huxingyi** [comment]
Please check it here:
https://github.com/huxingyi/dust3d/blob/1.0.0-rc.6/dust3d.pro#L537

I used the source files: MassSpringSolver.h and MassSpringSolver.cpp

Since the fast mass spring library will be removed in the next version from Dust3D, please drop discussion to the library repo if you still have issues. Thanks.

**Reply by aapatre** [reply]
Thanks a lot for your quick response! :)
